src/main/java/uk/gov/ch/model/update/ManagingOfficerData.java

```diff
@@ -0,0 +1,72 @@
+package uk.gov.ch.model.update;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@Entity
+public class ManagingOfficerData {
+    @Id
+    @Column(name = "MO_ID")
+    @JsonIgnore
+    private String managingOfficerId;
+
+    @Column(name = "RESIDENTIAL_ADDRESS")
+    @JsonProperty("residential_address")
+    private String residentialAddress;
```

Comment:
Address needs to be the fields from the sql query. We should use field names from: https://companieshouse.atlassian.net/wiki/spaces/ROE2/pages/4098261320/Public+Data+BO+Mapping#Adress-Mapping